### PR TITLE
Added 3 new command line options

### PR DESCRIPTION
**Pull request by TomSmartBishop**
--silent: suppress any command line output
--verbose: a lot of command line output
--out: specify a new output directory so that the rendered templates can be generated in a different location (reconstructing the relative directory structure)

This time I used autopep8 to fix formatting issues :)

### CODE REVIEW

**Comment by filwaitman** [comment]
@TomSmartBishop ok, thanks!  =]

I'll take a look in the next few hours!

File: README.rst [hunk 1]
```diff
@@ -29,15 +29,33 @@ Usage:
...
 
-For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
+* For a more intense usage, you can also use a settings file to define a bunch of things. To use settings file:
 
-.. code:: bash
+    .. code:: bash
+
+        jinja2_standalone_compiler <path-to-jinja-files> --settings=path/to/settings.py
+
+    Please note that settings file is a Python file, so Python syntax applies. =P
```
Comment (filwaitman) [inline README.rst]: This is being rendered as code on RST as well. Take a look:

screenshot from 2016-08-31 19-42-21

File: jinja2_standalone_compiler/__init__.py [hunk 2]
```diff
@@ -14,86 +14,123 @@
...
+        if out_path:
+            rel_path = os.path.relpath(jinja_template, path)
+            template_file = os.path.join(out_path, rel_path)
+            template_dir = os.path.dirname(template_file)
+            if not os.path.exists(template_dir):
+                try:
+                    os.makedirs(template_dir)
+                except:
+                    raise IOError(
+                        u'Cannot create sub ouput directory: {}'.format(template_dir))
```
Comment (filwaitman) [inline jinja2_standalone_compiler/__init__.py]: Typo on `ouput`

File: jinja2_standalone_compiler/__init__.py [hunk 3]
```diff
@@ -105,4 +142,12 @@ def main_command(path, settings=None):
         settings = imp.load_source(current_dir, settings)
 
     current_dir = os.path.join(current_dir, path)
-    main(settings=settings, path=current_dir)
+
+    if out and not os.path.exists(out):
+        try:
+            os.makedirs(out)
+        except:
+            raise IOError(u'Cannot create ouput directory: {}'.format(out))
```
Comment (filwaitman) [inline jinja2_standalone_compiler/__init__.py]: Typo on `ouput`

**Comment by filwaitman** [comment]
@TomSmartBishop Reviewed. Please take a look at my comments.

Also, please run `autopep8 --max-line-length 130` (79 chars as line length is obsolete nowadays.  :laughing: )

File: README.rst (hunk 1 above)
Comment (TomSmartBishop) [inline README.rst]: Doesn't look like code, but got an indent (which was intended). Couldn't find a nice way of doing that... ideas welcome :)

File: jinja2_standalone_compiler/__init__.py (hunk 2 above)
Comment (TomSmartBishop) [inline jinja2_standalone_compiler/__init__.py]: right

File: jinja2_standalone_compiler/__init__.py (hunk 3 above)
Comment (TomSmartBishop) [inline jinja2_standalone_compiler/__init__.py]: right

File: README.rst (hunk 1 above)
Comment (filwaitman) [inline README.rst]: In fact, it doesn't look like code.  :laughing: 

I'll use nested bullets here instead. Just tested on git and rendered as it should.
I'll make this change.

**Comment by filwaitman** [comment]
Sounds perfect! Thanks dude!

**Pull request merged**